Index: kdav/autotests/CMakeLists.txt
===================================================================
--- kdav.orig/autotests/CMakeLists.txt
+++ kdav/autotests/CMakeLists.txt
@@ -18,15 +18,3 @@ ecm_add_test(davurltest.cpp
     NAME_PREFIX "kdav-"
     LINK_LIBRARIES KPim::KDAV Qt5::Test Qt5::Core
 )
-
-ecm_add_test(davitemfetchjobtest.cpp fakeserver.cpp
-    TEST_NAME davitemfetchjob
-    NAME_PREFIX "kdav-"
-    LINK_LIBRARIES KPim::KDAV Qt5::Test Qt5::Core Qt5::Network
-)
-
-ecm_add_test(davitemslistjobtest.cpp fakeserver.cpp
-    TEST_NAME davitemslistjob
-    NAME_PREFIX "kdav-"
-    LINK_LIBRARIES KPim::KDAV Qt5::Test Qt5::Core Qt5::Network
-)
